packages/apps/stock/src/components/organisms/CardStockIndicator/CardStockIndicator.tsx

```diff
@@ -43,13 +36,16 @@ const CardStockIndicator = ({title, number}: CardStockIndicatorProps) => {
 
   return (
     <Card style={styles.card}>
-      <PopUpOneButton
+      <Alert
         visible={popUp}
-        data={formatNumber(number)}
         title={title}
-        btnTitle={I18n.t('Auth_Close')}
-        onPress={() => setPopUp(!popUp)}
-      />
+        confirmButtonConfig={{
+          width: 50,
+          title: null,
+          onPress: () => setPopUp(!popUp),
+        }}>
```

Comment:
Please use the times icon here to close the alert
Reduce alert width
Increase content text size